test/evergreen.yml

```diff
@@ -184,18 +184,23 @@ functions:
         set -o errexit
         set -o verbose
         pushd wiredtiger
-        mongo_repo=https://github.com/mongodb/mongo
         # Find the equivalent mongodb branch that matches the same branch version to wiredtiger. In
         # the case of patch builds, it is possible that the developer has a branch name that doesn't
         # adhere to naming rules, therefore derive the base branch name from git.
         branch=$(git branch --contains | grep -e "mongodb" -e "develop" -e "evg-pr-test-")
         if [[ $branch =~ "mongodb-" ]]; then
+          mongo_repo=https://github.com/mongodb/mongo
           mongo_branch=v$(echo $branch | cut -d'-' -f 2)
+          git clone $mongo_repo -b $mongo_branch
         else
-          mongo_branch=master
+          # FIXME-WT-14185 Use the public main repo when it is starting to sync with private repo.
+          # The WT-13948 ticket requires changes from the private repository. As a workaround
+          # use a mongodb forked repo with custom branch.
+          forked_repo=https://github.com/jiechenbo/mongo
+          mongo_branch=wt-13948-workaround
+          git clone $mongo_repo -b $mongo_branch
         fi
         popd
-        git clone $mongo_repo -b $mongo_branch
```

Comment:
You can leave this line here and assign `mongo_repo` and `mongo_branch` in the `if/else`

Reply:
[b26e53c](https://github.com/wiredtiger/wiredtiger/pull/11628/commits/b26e53cfac266dfb68efff1893b57f90356a2016)